# HG changeset patch
# User Júlio Hoffimann Mendes <julio.hoffimann@gmail.com>
# Date 1383428330 25200
#      Sat Nov 02 14:38:50 2013 -0700
# Node ID a5c4efa2e5a3bf5d6c9fb67d1e3fe37a2f03ac46
# Parent  f22714d4a5da540aebfffc150d8a8107f052f377
pt_BR.ts: Update Octave Portugese translation.

* libgui/languages/pt_BR.ts: Update Octave Portugese translation.

diff --git a/libgui/languages/pt_BR.ts b/libgui/languages/pt_BR.ts
--- a/libgui/languages/pt_BR.ts
+++ b/libgui/languages/pt_BR.ts
@@ -39,58 +39,58 @@
     <message>
         <location line="+1"/>
         <source>persistent</source>
         <translation>persistente</translation>
     </message>
     <message>
         <location filename="../qterminal/libqterminal/QTerminal.cc" line="+64"/>
         <source>foreground</source>
-        <translation type="unfinished"></translation>
+        <translation>plano de frente</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>background</source>
-        <translation type="unfinished"></translation>
+        <translation>plano de fundo</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>selection</source>
-        <translation type="unfinished"></translation>
+        <translation>seleçao</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>cursor</source>
-        <translation type="unfinished"></translation>
+        <translation>cursor</translation>
     </message>
 </context>
 <context>
     <name>QTerminal</name>
     <message>
         <location filename="../qterminal/libqterminal/QTerminal.h" line="+116"/>
         <source>Copy</source>
-        <translation type="unfinished">Copiar</translation>
+        <translation>Copiar</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Paste</source>
-        <translation type="unfinished"></translation>
+        <translation>Colar</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Clear All</source>
-        <translation type="unfinished"></translation>
+        <translation>Limpar Tudo</translation>
     </message>
 </context>
 <context>
     <name>QWinTerminalImpl</name>
     <message>
         <location filename="../qterminal/libqterminal/win32/QWinTerminalImpl.cpp" line="+1527"/>
         <source>copied selection to clipboard</source>
-        <translation type="unfinished"></translation>
+        <translation>selecao copiada para a prancheta</translation>
     </message>
 </context>
 <context>
     <name>documentation_dock_widget</name>
     <message>
         <location filename="../src/documentation-dock-widget.cc" line="+34"/>
         <source>Documentation</source>
         <translation>Documentação</translation>
@@ -177,17 +177,17 @@ is already open in the editor</source>
     <message>
         <location line="+4"/>
         <source>Cu&amp;t</source>
         <translation>Recor&amp;tar</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Paste</source>
-        <translation>Co&amp;lar</translation>
+        <translation>Colar</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>&amp;Next Bookmark</source>
         <translation>&amp;Próximo Marcador</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -237,108 +237,114 @@ is already open in the editor</source>
     <message>
         <location line="+63"/>
         <source>&amp;Recent Editor Files</source>
         <translation>Arquivos &amp;recentes</translation>
     </message>
     <message>
         <location line="+9"/>
         <source>&amp;Edit Function</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Editar Função</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>&amp;Close</source>
         <translation>&amp;Fechar</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Close All</source>
-        <translation>Fechar &amp;Todos</translation>
+        <translation>Fechar Todos</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Close Other Files</source>
         <translation>Fechar Outros Arquivos</translation>
     </message>
     <message>
         <location line="+49"/>
         <source>Run &amp;Selection</source>
-        <translation type="unfinished"></translation>
+        <translation>Roda &amp;Seleção</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>&amp;Help</source>
-        <translation type="unfinished">&amp;Ajuda</translation>
+        <translation>&amp;Ajuda</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>&amp;Help on Keyword</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Ajuda na Palavra-chave</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>&amp;Documentation on Keyword</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Documentação na Palavra-chave</translation>
     </message>
     <message>
         <location line="-147"/>
         <source>&amp;Find and Replace</source>
-        <translation>Procurar e Substituir</translation>
+        <translation>&amp;Procurar e Substituir</translation>
     </message>
     <message>
         <location line="-584"/>
         <source>Could not open file
 %1
 for read: %2.</source>
-        <translation type="unfinished"></translation>
+        <translation>Não foi possível abrir aquivo
+%1
+para leitura: %2.</translation>
     </message>
     <message>
         <location line="+21"/>
         <source>File
 %1
 does not exist. Do you want to create it?</source>
-        <translation type="unfinished"></translation>
+        <translation>Arquivo
+%1
+não existe. Deseja criá-lo?</translation>
     </message>
     <message>
         <location line="+17"/>
         <source>Could not open file
 %1
 for write: %2.</source>
-        <translation type="unfinished"></translation>
+        <translation>Não foi possível abrir arquivo
+%1
+para escrita: %2.</translation>
     </message>
     <message>
         <location line="+549"/>
         <source>Save File And Run</source>
         <translation>Salvar e Executar</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Go&amp;to Line</source>
-        <translation>Vá para Linha</translation>
+        <translation>Vá para &amp;Linha</translation>
     </message>
     <message>
         <location line="+53"/>
         <source>&amp;File</source>
         <translation>&amp;Arquivo</translation>
     </message>
     <message>
         <location line="+39"/>
         <source>&amp;Edit</source>
         <translation>&amp;Editar</translation>
     </message>
     <message>
         <location line="+22"/>
         <source>&amp;Preferences</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Preferências</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>&amp;Styles Preferences</source>
-        <translation type="unfinished"></translation>
+        <translation>Preferências de &amp;Estilo</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>&amp;Debug</source>
         <translation>&amp;Depurar</translation>
     </message>
     <message>
         <location line="+9"/>
@@ -995,37 +1001,37 @@ para:</translation>
         <location line="-1026"/>
         <location line="+946"/>
         <source>Save Workspace As</source>
         <translation>Salvar Ambiente de Trabalho como</translation>
     </message>
     <message>
         <location line="-826"/>
         <source>The release notes file &apos;%1&apos; is empty.</source>
-        <translation type="unfinished"></translation>
+        <translation>O arquivo de anotações da versão &apos;%1&apos; está vazio.</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>The release notes file &apos;%1&apos; cannot be read.</source>
-        <translation type="unfinished"></translation>
+        <translation>O arquivo de anotações da versão &apos;%1&apos; não pode ser lido.</translation>
     </message>
     <message>
         <location line="+13"/>
         <source>Octave Release Notes</source>
-        <translation type="unfinished"></translation>
+        <translation>Anotações da Versão do Octave</translation>
     </message>
     <message>
         <location line="+157"/>
         <source>Set working directory</source>
         <translation>Alterar diretório de trabalho</translation>
     </message>
     <message>
         <location line="+736"/>
         <source>Clear Clipboard</source>
-        <translation type="unfinished"></translation>
+        <translation>Limpar Prancheta</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Find Files...</source>
         <translation>Encontrar Arquivos...</translation>
     </message>
     <message>
         <location line="+5"/>
@@ -1075,77 +1081,77 @@ para:</translation>
     <message>
         <location line="+48"/>
         <source>Show File Browser</source>
         <translation>Mostrar Navegador de Arquivos</translation>
     </message>
     <message>
         <location line="+12"/>
         <source>Show News Window</source>
-        <translation type="unfinished"></translation>
+        <translation>Mostrar Janela de Notícias</translation>
     </message>
     <message>
         <location line="+11"/>
         <source>File Browser</source>
         <translation>Navegador de Arquivos</translation>
     </message>
     <message>
         <location line="+12"/>
         <source>News</source>
-        <translation type="unfinished"></translation>
+        <translation>Notícias</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Reset Default Window Layout</source>
         <translation>Recuperar Disposição de Janelas Padrão</translation>
     </message>
     <message>
         <location line="+86"/>
         <source>Octave Packages</source>
-        <translation type="unfinished"></translation>
+        <translation>Pacotes do Octave</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Share Code</source>
-        <translation type="unfinished"></translation>
+        <translation>Compartilhar o Código</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Contribute to Octave</source>
-        <translation type="unfinished"></translation>
+        <translation>Contribuir com o Octave</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Octave Developer Resources</source>
-        <translation type="unfinished"></translation>
+        <translation>Recursos para Desenvolvedores</translation>
     </message>
     <message>
         <location line="+32"/>
         <source>On Disk</source>
         <translation>No Disco</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Online</source>
         <translation>Online</translation>
     </message>
     <message>
         <location line="+12"/>
         <source>&amp;News</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Notícias</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Release Notes</source>
-        <translation type="unfinished"></translation>
+        <translation>Anotações da Versão</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Community News</source>
-        <translation type="unfinished"></translation>
+        <translation>Notícias da Comunidade</translation>
     </message>
     <message>
         <location line="+30"/>
         <source>Enter directory name</source>
         <translation>Informe nome do diretório</translation>
     </message>
     <message>
         <location line="+8"/>
@@ -1234,17 +1240,17 @@ para:</translation>
         <translation>Reportar Bug</translation>
     </message>
 </context>
 <context>
     <name>news_dock_widget</name>
     <message>
         <location filename="../src/news-dock-widget.cc" line="+44"/>
         <source>Community News</source>
-        <translation type="unfinished"></translation>
+        <translation>Notícias da Comunidade</translation>
     </message>
 </context>
 <context>
     <name>octave_dock_widget</name>
     <message>
         <location filename="../src/octave-dock-widget.cc" line="+56"/>
         <location line="+159"/>
         <source>Undock widget</source>
@@ -1261,32 +1267,32 @@ para:</translation>
         <translation>Acoplar widget</translation>
     </message>
 </context>
 <context>
     <name>octave_qscintilla</name>
     <message>
         <location filename="../src/m-editor/octave-qscintilla.cc" line="+127"/>
         <source>Help on</source>
-        <translation type="unfinished"></translation>
+        <translation>Ajuda em</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Documentation on</source>
-        <translation type="unfinished"></translation>
+        <translation>Documentação em</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Edit</source>
-        <translation type="unfinished"></translation>
+        <translation>Editar</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>&amp;Run Selection</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Executar Seleção</translation>
     </message>
 </context>
 <context>
     <name>octave_qt_link</name>
     <message>
         <location filename="../src/octave-qt-link.cc" line="+270"/>
         <source>The file %1 does not exist in the load path.  To debug the function you are editing, you must either change to the directory %2 or add that directory to the load path.</source>
         <translation>O arquivo %1 não existe no path. Para depurar a função, você precisa mudar para o diretório %2 ou adicionar esse diretório ao path.</translation>
@@ -1422,17 +1428,17 @@ para:</translation>
     <message>
         <location line="+61"/>
         <source>Restore editor tabs from previous session on startup</source>
         <translation>Reabrir abas da sessão anterior durante inicialização</translation>
     </message>
     <message>
         <location line="+13"/>
         <source>Create non existing files without prompting</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>
     </message>
     <message>
         <location line="+47"/>
         <source>Use custom file editor</source>
         <translation>Usar editor de texto externo</translation>
     </message>
     <message>
         <location line="+10"/>
@@ -1619,27 +1625,27 @@ para:</translation>
         <location line="+4"/>
         <location line="+346"/>
         <source>System setting</source>
         <translation>Configuração do sistema</translation>
     </message>
     <message>
         <location line="-279"/>
         <source>IBeam Cursor</source>
-        <translation type="unfinished"></translation>
+        <translation>Cursor IBeam</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Block Cursor</source>
-        <translation type="unfinished"></translation>
+        <translation>Cursor em Bloco</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Underline Cursor</source>
-        <translation type="unfinished"></translation>
+        <translation>Cursor Sublinhado</translation>
     </message>
     <message>
         <location line="+140"/>
         <source>Difference to the default size</source>
         <translation>Diferença do tamanho padrão</translation>
     </message>
     <message>
         <location line="+6"/>
@@ -1693,38 +1699,49 @@ para:</translation>
     <message>
         <location line="+14"/>
         <source>Welcome to Octave!</source>
         <translation>Bem-vindo ao Octave!</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>You seem to be using the Octave graphical interface for the first  time on this computer.  Click &apos;Finish&apos; to write a configuration file  and launch Octave GUI.</source>
-        <translation type="unfinished"></translation>
+        <translation>Parece que você está a interface gráfica do Octave pela primeira vez neste computador. Clique &apos;Finalizar&apos; para escrever o arquivo de configuração e iniciar o Octave.</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>The configuration file is stored in %1. If that file exists, you will not see this dialog when Octave starts again.</source>
-        <translation type="unfinished"></translation>
+        <translation>O arquivo de configuração é armazenado em %1. Se o arquivo existir, você não verá este diálogo quando o Octave reiniciar.</translation>
     </message>
     <message>
         <location line="+23"/>
         <source>&lt;html&gt;&lt;head&gt;
 &lt;style&gt;
 a:link { text-decoration: underline; color: #0000ff; }
 &lt;/style&gt;
 &lt;head/&gt;&lt;body&gt;
 &lt;p&gt;For more information about Octave:&lt;/p&gt;
 &lt;ul&gt;
 &lt;li&gt;Visit &lt;a href=&quot;http://octave.org&quot;&gt;http://octave.org&lt;/a&gt;&lt;/li&gt;
 &lt;li&gt;Get the documentation online as &lt;a href=&quot;http://www.gnu.org/software/octave/doc/interpreter/index.html&quot;&gt;html&lt;/a&gt;- or &lt;a href=&quot;http://www.gnu.org/software/octave/octave.pdf&quot;&gt;pdf&lt;/span&gt;&lt;/a&gt;-document&lt;/li&gt;
 &lt;li&gt;Open the documentation browser of Octave GUI with the help menu&lt;/li&gt;
 &lt;/ul&gt;
 &lt;/body&gt;&lt;/html&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;html&gt;&lt;head&gt;
+&lt;style&gt;
+a:link { text-decoration: underline; color: #0000ff; }
+&lt;/style&gt;
+&lt;head/&gt;&lt;body&gt;
+&lt;p&gt;Para mais informações sobre o Octave:&lt;/p&gt;
+&lt;ul&gt;
+&lt;li&gt;Visite &lt;a href=&quot;http://octave.org&quot;&gt;http://octave.org&lt;/a&gt;&lt;/li&gt;
+&lt;li&gt;Veja a documentação online em document &lt;a href=&quot;http://www.gnu.org/software/octave/doc/interpreter/index.html&quot;&gt;html&lt;/a&gt;- or &lt;a href=&quot;http://www.gnu.org/software/octave/octave.pdf&quot;&gt;pdf&lt;/span&gt;&lt;/a&gt;&lt;/li&gt;
+&lt;li&gt;Abra a documentação do Octave no menu de ajuda&lt;/li&gt;
+&lt;/ul&gt;
+&lt;/body&gt;&lt;/html&gt;</translation>
     </message>
     <message>
         <location line="+20"/>
         <source>Finish</source>
         <translation>Finalizar</translation>
     </message>
 </context>
 <context>
